mysql/lib/dialect/mysql2_driver_dialect.ts

```diff
@@ -52,4 +54,27 @@ export class MySQL2DriverDialect implements DriverDialect {
   getAwsPoolClient(props: PoolOptions): AwsPoolClient {
     return new AwsMysqlPoolClient(props);
   }
+
+  setupInitialNetworkTimeouts(props: Map<string, any>) {
+    this.setConnectTimeout(props);
+  }
+
+  setConnectTimeout(props: Map<string, any>) {
+    const wrapperConnectTimeout = props.get(WrapperProperties.WRAPPER_CONNECT_TIMEOUT.name);
+
+    if (wrapperConnectTimeout) {
+      props.set(MySQL2DriverDialect.connectTimeoutPropertyName, wrapperConnectTimeout);
+    }
+  }
+
+  setQueryTimeout(props: Map<string, any>, sql?: any) {
+    const queryTimeout = props.get(WrapperProperties.WRAPPER_QUERY_TIMEOUT.name);
+    if (queryTimeout && !sql[this.getQueryTimeoutPropertyName()]) {
+      sql[this.getQueryTimeoutPropertyName()] = queryTimeout;
+    }
+  }
+
+  getQueryTimeoutPropertyName(): string {
```

Comment:
Curious if this method is required if the property name is only used locally now.